installation/alpine.sh

```diff
@@ -0,0 +1,591 @@
+#/bin/ash
+
+# All-in-one installation without Docker
+
+immich_ver=1.27.0_37-dev
+alpine_ver=3.14
+
+tmp_dir=/tmp
+
+# see parse_args()
+# [0|1]
+dev=0
+
+# Parse arguments given to the script from the CLI.
+# Show help message, run installation and run install in dev mode.
+#
+# params:
+# - $@: args from cli
+#
+# return: 
+# - exit 0 if shows help message
+parse_args()
+{
+    if [ "$1" == "--install" ]
+    then
+        # Instead of downloading code from a release,
+        # the source code is provided.
+        if [ "$2" == "--dev" ]
+        then
+            dev=1
+        elif [ "$2" == "-h" ]
+        then
+            echo "Dockerless installation script." 
+            echo "usage: $0 --install [--dev|-h]"        
+            echo                                           
+            echo "  -h              Show this help message"
+            echo "  --dev           Run in dev mode"
+            echo  
+            exit 0
+        fi
+    else
+        echo "Dockerless installation script."         
+        echo "usage: $0 [--install] [--dev|-h]"
+        echo                                           
+        echo "  --install       Install Immich"
+        echo "  -h              Show this help message"
+        echo "  --dev           Run in dev mode"
+        echo                                         
+        exit 0
+    fi
+}
+
+# Shows a message in a box, which looks like:
+#
+# ###########
+# # Message #
+# ###########
+#
+# params:
+# - $1: message to print
+#
+# return:
+# - '1' if message is empty 
+display_message_box()
+{
+    # Calculate message and box width
+    message_length=$(echo -n "$1" | wc -c)
+    width=$((message_length + 4))
+
+    # Leave if message is empty
+    if [ -z "$message_length" ]
+    then
+        return 1
+    fi
+
+    echo
+
+    # Print top box row
+    for i in $(seq ${width})
+    do
+        echo -n "#"
+    done
+    echo
+
+    # Print message with borders
+    echo -n "# "; echo -n "$1" ; echo " #"
+
+    # Print bottom box row
+    for i in $(seq ${width})
+    do
+        echo -n "#"
+    done
+    echo
+}
+
+# Check OS and version are supported.
+#
+# params: nothing
+#
+# return:
+# - exit 1 if OS isn't supported
+# - exit 2 OS version isn't supported
+check_os()
+{
+	os_ver_file=/etc/alpine-release
+
+	if [ ! -e "$os_ver_file" ]
+	then
+		echo "OS not supported! Please run it on Alpine Linux."
+		exit 1
+	fi
+
+	if ! grep -qE "^$alpine_ver" "$os_ver_file"
+	then
+		echo "Alpine $(cat $os_ver_file) not supported: need Alpine $needed_os_ver."
```

Comment:
you forgot to rename this var `needed_os_ver`

Reply:
thanks